language/move-stdlib/sources/fixed_point32.move

```diff
@@ -154,6 +154,136 @@ module std::fixed_point32 {
         num.value == 0
     }
 
+    /// Returns the smaller of the two FixedPoint32 numbers.
+    public fun min(num1: FixedPoint32, num2: FixedPoint32): FixedPoint32 {
```

Comment:
If the values are equal then this will return num2. This deviates from [Rust's](https://doc.rust-lang.org/std/cmp/fn.min.html
) and [C++'s](https://en.cppreference.com/w/cpp/algorithm/min) definition of min.

Reply:
@emmazzz @junkil-park does this matter to you? I can switch the two, although the content will be the same anyway.

Reply:
Doesn't matter to me.

Reply:
Yeah this shouldn't matter as long as the contents are the same.